account_sponsor_page.php

```diff
@@ -200,7 +200,7 @@
 		# summary
 		echo '<td>' . string_display_line( $t_bug->summary );
 		if( VS_PRIVATE == $t_bug->view_state ) {
-			printf( ' <i class="fa fa-lock" alt="(%s)" title="%s"></i>', lang_get( 'private' ), lang_get( 'private' ) );
```

Comment:
Because it is not in <img> context. And I doubt you'll ever see 'alt' value in <i> context in any existing browser.